src/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_Helpers.cs

```diff
@@ -19,14 +20,14 @@ struct NonBlittableType
         public string s;
     }
 
-    private static ThreadPoolBoundHandle CreateThreadPoolBoundHandle()
+    private ThreadPoolBoundHandle CreateThreadPoolBoundHandle()
     {
         return CreateThreadPoolBoundHandle((SafeHandle)null);
     }
 
-    private static ThreadPoolBoundHandle CreateThreadPoolBoundHandle(SafeHandle handle)
+    private ThreadPoolBoundHandle CreateThreadPoolBoundHandle(SafeHandle handle)
     {
-        handle = handle ?? HandleFactory.CreateAsyncFileHandleForWrite();
+        handle = handle ?? HandleFactory.CreateAsyncFileHandleForWrite(GetTestFilePath());
```

Comment:
Nit: consider adding the same [Caller_] arguments to CreateThreadPoolBoundHandle and then passing those through to GetTestFilePath.  Then the file path will include info on the test using this.